frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -696,8 +712,33 @@
                          :on-accept add-color
                          :data {:color "#406280"
                                 :opacity 1}
-                         :position :right})))]
-    [:div.asset-section
+                         :position :right})))
+
+        unselect-all
+        (mf/use-callback
+          (fn [event]
+            (swap! state assoc :selected empty-selection)))
+
+        on-select
+        (mf/use-callback
+          (mf/deps state)
```

Comment:
do you need to depend on `state`? you're not reading nothing. I think you don't need dependencies here

Reply:
I read the state to make a swap and update

Reply:
The state on deps is not necessary